docs/usage/configuration/config_documentation.md

```diff
@@ -3815,14 +3815,15 @@ Sub-options for this setting include:
 * `system_mxid_display_name`: set the display name of the "notices" user
 * `system_mxid_avatar_url`: set the avatar for the "notices" user
 * `room_name`: set the room name of the server notices room
-
+* `auto_join`: the user will be automatically joined to the room instead of being invited
```

Comment:
Please add what version of Synapse this was added in (and note that it defaults to False)